components/Service/Service.jsx

```diff
@@ -0,0 +1,136 @@
+import { gsap } from 'gsap'
+import Image from 'next/image'
+import { useEffect, useState } from 'react'
+
+import ServiceCard from '@/components/Service/ServiceCard'
+
+import { navigationList } from '@/data/service'
+import useConvert from '@/hooks/useConvert'
+import { pxToVw } from '@/utils/style'
+
+const Service = () => {
+  const { isSp } = useConvert()
+  const [tl] = useState(gsap.timeline({ paused: true }))
+  const [pageCount, setPageCount] = useState(0)
+
+  const nextPage = () => {
+    tl.play()
+    setPageCount(++pageCount)
+  }
+  const backPage = () => {
+    tl.reverse()
+    setPageCount(--pageCount)
+  }
+
+  const jumpPage = (index) => {
+    setPageCount(index)
+    tl.pause(index * 0.8)
+  }
+
+  useEffect(() => {
+    const items = gsap.utils.toArray('div.service-card-element')
+
+    items.forEach((item, index) => {
+      gsap.set(item, { x: isSp ? 10 * index : 30 * index })
+      if (index !== 4) {
+        tl.to(item, {
+          x: -100,
+          y: 1,
+          opacity: 0,
+          duration: 0.4,
+        })
+          .to(items[index + 1], {
+            x: 0,
+            duration: 0.4,
+          })
+          .addPause()
+      }
+    })
+  }, [isSp])
+
+  return (
+    <div>
+      <div
+        className="relative mt-[6.9vw]"
+        style={{
+          width: pxToVw(250, 935),
+          height: pxToVw(200, 382),
+          left: pxToVw(0, -190),
+        }}
+      >
+        <ServiceCard />
+        <div
+          className="absolute flex w-full items-center justify-center"
+          style={{
+            bottom: pxToVw(10, 20),
+          }}
+        >
+          <button
+            className="z-10 mr-3 cursor-pointer rounded-full bg-[#D9D9D9]"
+            style={{
+              width: pxToVw(20, 40),
+              height: pxToVw(20, 40),
+              padding: pxToVw(4, 8),
+            }}
+            onClick={nextPage}
+            disabled={pageCount === 4}
+          >
+            <Image
+              src="/img/angles_left.svg"
+              width={20}
+              height={20}
+              layout="responsive"
+              alt=""
+            />
+          </button>
+          {navigationList.map((_, index) => (
```

Comment:
@Shu1512 
ここで、[1~5]が入っている配列を回すという設計にすると、serviceListsが増えても下のpaginationは増えないですよね、、？
理想はserviceListsの数に応じて下のpaginationが増減する、だと思います。

Reply:
自分のサイトでもやってるけど、下記の様に修正するといけるかもしれません！
```suggestion
          {[...Array(navigationList.length)].map()
```